server/chat-commands/moderation.ts

```diff
@@ -873,7 +873,12 @@ export const commands: ChatCommands = {
 		}
 
 		this.privateGlobalModAction(`${user.name} unlocked the ${range ? "IP range" : "IP"}: ${target}`);
-		this.globalModlog(`UNLOCK${range ? 'RANGE' : 'IP'}`, target, ` by ${user.name}`);
+		(room || Rooms.global).modlog({
+			action: `UNLOCK${range ? 'RANGE' : 'IP'}`,
+			ip: target,
+			isGlobal: true,
+			loggedBy: user.id,
+		});
 	},
```

Comment:
Why does this have such a different function signature than `globalModlog`?

Reply:
because `globalModlog` doesn't support logging a random IP address not associated with the user and since we only need to do that in one place I didn't think a special case in `globalModlog` was worth it